oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/CachingDataStoreTest.java

```diff
@@ -638,4 +642,165 @@ private void waitFinish() {
             e.printStackTrace();
         }
     }
+    private AbstractSharedCachingDataStore createDataStore() throws Exception {
```

Comment:
Can we not use the existing dummy objects to construct the datastore. For the variables we can create instance variables instead of static variables to make it easier to inject.

Reply:
@amit-jain I don't know what I was thinking when using static variables for the recordCache configuration, sorry for that. I'm using the existing test datastore now, but I had to adapt it a bit for simulating a delay when doing (remote) backend requests. Could you have another look? Thank you very much!